bundles/org.openhab.binding.opensprinkler/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -89,6 +89,9 @@
 			<channel id="resetStations" typeId="resetStations"></channel>
 			<channel id="enablePrograms" typeId="enablePrograms"></channel>
 			<channel id="rainDelay" typeId="rainDelay"></channel>
+			<channel id="queuedZones" typeId="queuedZones"></channel>
+			<channel id="cloudConnected" typeId="cloudConnected"></channel>
+			<channel id="pausePrograms" typeId="pausePrograms"></channel>
```

Comment:
Please provide thing upgrade instructions (for an automatic adding of these new channels to existing things created with Main UI).